models/src/main/java/com/vimeo/networking2/ApiError.kt

```diff
@@ -10,44 +10,32 @@ import com.vimeo.networking2.enums.asEnum
 /**
  * This class represents an error response from the Vimeo API. It holds useful getters to
  * understand why your request might have failed.
+ *
+ * @param developerMessage Developer friendly error message.
+ * @param userMessage User friendly error message, can be used for displaying messages to the user.
+ * @param errorMessage User friendly error message.
```

Comment:
I figured out I can remove `userMessage` as the API doesn't return it after all. There are two scenarios here
1. The API gives us a real basic error and only `errorMessage` is populated. The message shouldn't be used for displaying to the user because it's probably not localized.
2. The API gives us an `errorMessage` and a `developerMessage`. In this case, the `errorMessage` is localized and can be shown to the user.

I'm updating the class with this info.